### PR TITLE
Make lighting more generic, fix Shader variable types, update textures

### PR DESCRIPTION
**Pull request by erich666**
Remove the light intensities so that default values are set by viewers. Fix color3f and normal3f to float3 as needed. Update textures to latest versions. README.md's discuss the changes and how to properly view the models in different viewers.

Inlcudes proposed PRs https://github.com/usd-wg/assets/pull/44 and https://github.com/usd-wg/assets/pull/51

### CODE REVIEW

**Comment by erich666** [comment]
See https://github.com/usd-wg/assets/pull/44#issuecomment-1692190349 for more about what various changes are about. All comments appreciated - I'm not an expert at USD, just trying to provide a reasonable model that provides various types of UsdPreviewSurfaces materials, notes issues about emission intensities for lights and surfaces, "double-sided" testing, and other areas.

**Comment by hybridherbst** [comment]
The new USDZ files contain your entire users path as part of the directory structure which I think you may want to change:

image

**Comment by jcowles** [comment]
Ah, good catch Felix

**Comment by jcowles** [comment]
As for structure, my preference is:

/McUsd (a new xform, under which everything is parented)
    /Geom (this is McUsd in the current asset)
    /Sun
    /DomeLight
    /Camera
    /Looks

**Comment by erich666** [comment]
@jcowles @hybridherbst OK, take a look, please.

File: test_assets/NormalsTextureBiasAndScale/screenshots/crn_usdview_light_from_above.png
Comment (erich666) [inline test_assets/NormalsTextureBiasAndScale/screenshots/crn_usdview_light_from_above.png:1]: Yes, these changed considerably, due to lighting definitions changing. More dramatic, but clearer, I think.

**Comment by jcowles** [comment]
LGTM, merging - thanks!!!

**Pull request merged**